cpp/daal/src/externals/service_service_mkl.h

```diff
@@ -37,26 +41,33 @@ namespace mkl
 {
 struct MklService
 {
-    static void * serv_malloc(size_t size, size_t alignment) { return fpk_serv_malloc(size, alignment); }
+    static void * serv_malloc(size_t size, size_t alignment) { return aligned_alloc(size, alignment); }
 
-    static void serv_free(void * ptr) { fpk_serv_free(ptr); }
+    static void serv_free(void * ptr) { free(ptr); }
 
-    static void serv_free_buffers() { fpk_serv_free_buffers(); }
+    static void serv_free_buffers() { mkl_free_buffers(); }
```

Comment:
Does it mean that MKL is able to release all the allocated memory at once? Does it result in "double free" errors?